### ISSUE TITLE
Change from Handlebar to Angle brackets in guides-source/guides/release/tutorial/subroutes.md

### ISSUE THREAD

**Issue by lisaychuang** [ember-learn/guides-source#507]
Hi learning team,
In the Subroutes section of the Tutorial, the code sample changed from Handlebar syntax (per earlier section "Routes & Templates") to Angle bracket syntax for components:

As the tutorial hasn't introduced angle bracket syntax, this inconsistency may be confusing to users learning Ember for the first time.

**Current code in `rentals.hbs` template:**
```js 
// app/templates/rentals.hbs

{{#list-filter
 filter=(action "filterByCity") as |filteredResults|
}}
  <ul class="results">
    {{#each filteredResults as |rentalUnit|}}
      <li>{{rental-listing rental=rentalUnit}}</li>
    {{/each}}
  </ul>
{{/list-filter}}
```

**Code sample shown in [Subroutes](https://guides.emberjs.com/release/tutorial/subroutes/)** as we begin to refactor code from main route to nested route:

```js
<ListFilter
   @filter={{action "filterByCity"}}
   as |filteredResults|}}
  <ul class="results">
    {{#each filteredResults as |rentalUnit|}}
      <li><RentalListing @rental={{rentalUnit}} /></li>
    {{/each}}
  </ul>
</ListFilter>
```

**Comment by locks** [comment]
Hi @lisaychuang! Thank you for your continued feedback 😁

I went through the tutorial to check syntax usage.
Although `link-to` is used in the classic syntax (for a couple specific reasons), the tutorial seems to use angle bracket invocation syntax for components on the whole, with an implicit introduction and an explicit Zoey callout at [Building a Simple Component](https://guides.emberjs.com/release/tutorial/simple-component/).

Could this inconsistency have been because we deployed an update while you were going through the tutorial?
